docs/vgclog_examples.md

```diff
@@ -706,3 +705,745 @@ Analyzing the structure and elements of this example log output shows that this
+
+```xml
+<cycle-start id="1154" type="global mark phase" contextid="0" timestamp="2021-02-26T11:17:25.034" intervalms="374365.075" />
+```
+
+The cycle begins with the STW subincrement of a global *mark* phase increment. The STW subincrement is recorded using the `<gc-start>` element of type `global mark phase`.
+
+```xml
+<gc-start id="1155" type="global mark phase" contextid="1154" timestamp="2021-02-26T11:17:25.034">
+  <mem-info id="1156" free="1442840576" total="4294967296" percent="33">
+    <remembered-set count="2197888" freebytes="162912768" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="130" regionsrebuilding="0"/>
+  </mem-info>
+</gc-start>
+
+```
+
+The `<gc-start>` element provides a snapshot of the free memory available in the heap and the status of marked objects. At the start of the increment, the heap is 33% free, with 1376 MB (1442840576 B) of the total 4096 MB (4294967296 B) heap free.
+
+The `<remembered-set>` element records the status of the remembered set metastructure of the JVM, a structure that records object references that cross different regions. During the rebuilding of the *remembered set* metastructure, any regions that cannot be rebuilt into a remembered set due to a lack of memory resource in the metastructure are marked as *overflow* regions. Cycles cannot reclaim memory from overflow regions.
+
+The aim of the global mark cycle to create a new record of object liveness can be seen in the remembered set statistics. The global mark cycle populates the remembered set and also attempts to completely rebuild the set for the overflowed regions. 
+
+The logs show that at the start of this STW subincrement, the remembered set count is 2,197,888 cards, the metastructure is using 94% of its total available memory, and three overflow regions need to be rebuilt.
+
+The `<gc-op>` element records that the STW subincrement runs a [mark operation](gc_overview.md#gc-mark-operation). This operation begins the process of building a record of object liveness across the heap.
+
+```xml
+<gc-op id="1157" type="mark increment" timems="122.825" contextid="1154" timestamp="2021-02-26T11:17:25.157">
+  <trace-info objectcount="7726701" scancount="7584109" scanbytes="213445656" />
+</gc-op>
+```
+
+The `<trace-info>` element records information about the marking and scanning stages of the mark increment operation. `objectcount` records the number of objects that have been marked, ready for tracing. After marking live objects, a scan is run to trace objects and references. `scancount` records the number of marked objects that have been scanned. `scanbytes` records the total memory of all marked objects that have been scanned.
```

Comment:
Are these 2 paragraphs ok now? - check the use of 'mark', 'trace', 'scan'. Does this agree with the definition of a mark operation in the gc_overview.md file?

Reply:
My try, but feel free to massage and shuffle it: 
`objectcount` records the number of objects that have been marked (determined as live), ready for later scanning what will find new object references and child objects. `scancount` records the number of scanned. `scanbytes` records the total memory of scanned objects. `scancount` is slightly lower than `objectcount` because 1) during tracing object scanning is slightly delayed relative to marking and 2) even when both marking and scanning completes, some marked objects that are known to be leafs (with no child objects) are not scanned at all.

Reply:
Marking and scanning are overlapping (in big picture, from perspective of the whole cycle). Although for a given object marking and scanning are apart in time. But those things interleave (one thread does a bit of marking work for some objects and then bit of scanning work for some other objects).  Big picture is more important - we cannot really say they are separate 'stages'. Better work is perhaps 'activity':

'The `<trace-info>` element records information about the marking and scanning activity'